finetune/util/text_generation.py

```diff
@@ -3,6 +3,34 @@
 from finetune.util.shapes import shape_list
 
 
+def format_mlm_predictions(gen_text_key, mlm_ids_key, mlm_positions_key):
```

Comment:
this seems to be a repeat of top_k_report?

Reply:
It is, I can pull it out of here--I have written some more generic formatting code in the branch I'm using for tensorboard stuff. I can combine that without a refactor of top_k_prediction. Will pull out the code in text generation for now.